Workbooks/Workloads/Collection logs/Collection logs.workbook

```diff
@@ -0,0 +1,81 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "f1e2d7f8-f4ff-4bf0-a178-93887b200447",
+            "version": "KqlParameterItem/1.0",
+            "name": "VirtualMachine",
+            "type": 5,
+            "isRequired": true,
+            "value": "value::1",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.compute/virtualmachines": true
+              },
+              "additionalResourceOptions": [
+                "value::1"
+              ]
+            },
+            "resourceType": "microsoft.insights/components"
+          }
+        ],
+        "style": "above",
+        "queryType": 0,
+        "resourceType": "microsoft.insights/components"
+      },
+      "name": "parameters - 0"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "### Collection logs from {VirtualMachine:name}"
+      },
+      "name": "text - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "telegraflogs_CL\n| order by TimeGenerated desc\n| extend Message = extract(@'.+Z D! \\[outputs.socket_writer\\] (.+)', 1, RawData) \n| project Time = TimeGenerated, Message = iff(Message == '', RawData, Message), ['Log type'] = iff(Type == 'telegraflogs_CL', 'Telegraf', 'WA Agent')\n| take 200",

```

Comment:
We also need logs from "waagentlogs_CL" table. Seems, this query is only getting logs from "telegraflogs_CL"

Reply:
I'll fix this in a separate change.